src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -107,46 +107,46 @@ struct TC_GAME_API AuctionEntry
 //this class is used as auctionhouse instance
 class TC_GAME_API AuctionHouseObject
 {
-  public:
```

Comment:
revert entire file, just remove indent from this line only

Reply:
i suppose private part too?